modules/@apostrophecms/module/index.js

```diff
@@ -602,6 +581,29 @@ module.exports = {
         self.action = `/api/v1/${self.__meta.name}`;
       },
 
+      async executeAfterModuleInitTask() {
+        for (const [ name, info ] of Object.entries(self.tasks || {})) {
+          if (info.afterModuleInit) {
+            // Execute a task like @apostrophecms/asset:build or
+            // @apostrophecms/db:reset which
+            // must run before most modules are awake
+            if (self.apos.argv._[0] === `${self.__meta.name}:${name}`) {
```

Comment:
See:

https://github.com/apostrophecms/apostrophe/pull/2650/files#diff-21afebea4dcabbb90a4ae6403ea2fd903171083d2699721e8fd07f5af745c382R195

Setting `afterModuleInit` when you configure a task means that the task will run immediately after its own module is initialized. If the module runs before the `@apostrophecms/db` module then that means before the database connection is made, otherwise not.

Reply:
Ok, makes sense